client/signup/steps/jpo-site-type/select-business-address.jsx

```diff
@@ -0,0 +1,82 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { translate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import Button from 'components/button';
+import FormLabel from 'components/forms/form-label';
+import FormTextInput from 'components/forms/form-text-input';
+
+class SelectBusinessAddress extends React.Component {
+
+	static propTypes = {
+		signupDependencies: PropTypes.object,
+		handleBusinessInfo: PropTypes.func,
+		businessInfo: PropTypes.object,
+		required: PropTypes.bool,
+	};
+
+	render() {
+		if ( ! this.props.current ) {
+			return ( <div /> );
+		}
+
+		const {
+			handleBusinessInfo,
+			businessInfo: {
+				businessName,
+				streetAddress,
+				city,
+				state,
+				zipCode
+			}
+		} = this.props;
+
+		return ( 
+			<div className="jpo__site-type-wrapper business-address">
+				<Card>
+					<FormLabel>{ translate( 'Business Name' ) }</FormLabel>
+					<FormTextInput
+						name="businessName"
+						value={ businessName }
+						onChange={ handleBusinessInfo }
+						/>
+					<FormLabel>{ translate( 'Street Address' ) }</FormLabel>
```

Comment:
Hi! I've found a possible matching string that has already been translated **32** times:
`translate( 'Invalid street address' )` *ES Score: 12*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/feature/jpo)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).